client/gribiclient.go

```diff
@@ -345,7 +347,23 @@ func (c *Client) Connect(ctx context.Context) error {
 				log.V(2).Infof("shutting down recv goroutine")
 				return
 			}
-			if done := respHandler(stream.Recv()); done {
+			resp, err := stream.Recv()
+			trailer := stream.Trailer()
+
+			if len(trailer) != 0 {
+				if url, ok := trailer["errordetails-typeurl"]; ok {
+					anyObj := &anypb.Any{TypeUrl: url[0], Value: []byte(trailer.Get("errordetails-value-bin")[0])}
```

Comment:
Can `url[0]` here be anything other than the gRIBI `ModifyErrorDetails` path? If not, should we just make this specifically handle this case?

Reply:
With current gRIBI definition, url[0] will only contain ModifyErrorDetails

Can you expand on "should we just make this specifically handle this case"

Reply:
Exactly, so is there a reason for us to not just say that this is the only valid type -- and not include the `errordetails-typeurl` in the metadata? Do we need the extensibility?